app/policies/search_policy.rb

```diff
@@ -0,0 +1,7 @@
+class SearchPolicy < ApplicationPolicy
+  attr_reader :user, :record
```

Comment:
remove this attr_reader, we have this in application policy, i already fixed all policies in my PR